doc/themes/scikit-learn/layout.html

```diff
@@ -275,7 +275,7 @@
       <p class="doc-version"><b>{{project}} v{{ release|e }}</b><br/>
       <a href="http://scikit-learn.org/stable/support.html#documentation-resources">Other versions</a></p>
     {%- endif %}
-    <p class="citing">Please <b><a href="about.html#citing-scikit-learn" style="font-size: 110%;">cite us </a></b>if you use the software.</p>
+    <p class="citing">Please <b><a href="{{ '{}#citing-scikit-learn'.format(pathto('about')).replace('##', '#') }}" style="font-size: 110%;">cite us </a></b>if you use the software.</p>
```

Comment:
This is really ugly, but otherwise the anchor link becomes `href="##citing-scikit-learn"` with a double `#` sign when you're already at `about.html`.